extensions/digitalocean/secrets.go

```diff
@@ -7,108 +7,36 @@ See the LICENSE file for more details.
 package digitalocean
 
 import (
-	"context"
-	"strings"
+	"fmt"
 
 	providerConfig "github.com/kubefirst/kubefirst-api/pkg/providerConfigs"
 	pkgtypes "github.com/kubefirst/kubefirst-api/pkg/types"
 	"github.com/rs/zerolog/log"
-
-	v1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 )
 
-func BootstrapDigitaloceanMgmtCluster(clientset *kubernetes.Clientset, cl *pkgtypes.Cluster, destinationGitopsRepoURL string) error {
-	err := providerConfig.BootstrapMgmtCluster(
-		clientset,
-		cl.GitAuth.User,
-		destinationGitopsRepoURL,
-		cl.GitProtocol,
-		cl.CloudflareAuth.Token,
```

Comment:
@jarededwards all of the other "extensions" have this variable set to `cl.CloudflareAuth.APIToken` but this one had it as `cl.CloudflareAuth.Token`. Should I keep my change? I set it back to `cl.CloudflareAuth.APIToken`.

Reply:
Solved internally. We want to have it fail if it's not APIToken. Fixed in https://github.com/kubefirst/kubefirst-api/pull/391/commits/d1d1c004deeac8c2c2b82db0a97e0236d0a2d3e9